examples/config.sample

```diff
@@ -98,8 +103,7 @@
 #Mavlink-router serves on this TCP port
 TcpServerPort=5790
 ReportStats=false
-#Which mavlink dialect is being used. Can be `common`[default] or `ardupilotmega`
-MavlinkDialect=ardupilotmega
+MavlinkDialect=auto
```

Comment:
unrelated. Either a separate commit or at least mention in the commit message that while changing this you fixed the default value here.